config/locales/en.yml

```diff
@@ -98,4 +98,8 @@ en:
     cookie-policy: Cookies
     preferences_saved_html: Your cookie settings were saved</br><a class='govuk-link' href='%{return_url}'>Go back to Help for early years providers</a>
 
+  content-block-content:
+    landing-page-introduction: <h1 class="govuk-heading-xl govuk-!-margin-bottom-4">Help for early years providers</h1><p class="govuk-body-l">The Early Years Foundation Stage (EYFS) has changed. Find guidance and practical support to help you with the changes</p><p class="govuk-body-l">These resources are for childminders, nursery leaders and pre-school practitioners.</p>
+    other_useful_resources: <h2 class="govuk-heading-l">Other useful resources</h2><p class="govuk-body"><a class="govuk-link" href="https://www.gov.uk/government/publications/changes-to-the-early-years-foundation-stage-eyfs-framework/changes-to-the-early-years-foundation-stage-eyfs-framework">Changes to the early years foundation stage framework</a></p><p class="govuk-body"><a class="govuk-link" href="https://www.gov.uk/government/publications/early-years-foundation-stage-framework--2">Statutory framework for the early years foundation stage</a></p><p class="govuk-body"><a class="govuk-link" href="https://www.gov.uk/government/publications/development-matters--2">Development Matters, non-statutory curriculum guidance for the early years foundation stage</a></p>
+
 
```

Comment:
this belongs with the other content blocks under /config/locales/content/blocks.yml

Adding the html is dead easy.

Reply:
Oh ok, I had not noticed that one already exists.   Good spot about the snake case, it should be underscore